### ISSUE TITLE
Alt-Shift-arrow keys no longer work in Chrome when extension is enabled

### ISSUE THREAD

**Issue by yang** [opus1269/photo-screen-saver#16]
Minor issue: on Macs, alt-shift-left/right selects the entire preceding/following word, but when this extension is enabled, that selection no longer works, including in the address bar.

**Comment by opus1269** [comment]
It's always something, haha.  It's really hard to find shortcut keys that aren't used somewhere on the browser itself or Windows, Mac, or Chrome OS. (especially Chrome OS which has a ton of them predefined.)

I can define the default shortcut keys by OS, but i was trying to avoid that. Do you know what would be a good modifier key combination for Mac with the left/right arrows? They have to have a Ctrl or Alt and can't have Ctrl+Alt Apparently, Mac also has a Command and a Ctrl. [These are the possibilities](https://developer.chrome.com/extensions/commands)

In the meantime, you can go to the extensions page, scroll to the bottom, click on "Keyboard shortcuts" and change them. If you don't use the interactive mode of the screensaver, you can delete the forward, back, and pause altogether.

**Comment by yang** [comment]
Oh thanks, didn't realize that's where this was coming from. Sure, I'll
think about what would be a good shortcut key.

**Comment by yang** [comment]
To simplify, improve discoverability and improve remember-ability, what about making just the arrow keys by themselves work (without modifiers)?

**Comment by opus1269** [comment]
Yes. That is the best solution and my original design. The problem was you can't listen for raw keyboard events on Chrome OS (I suppose it is a security thing to protect against keyloggers).

I will bite the bullet and do an OS dependent implementation of this feature.

Thanks for your input.
